frontend/src/views/teamInfo.js

```diff
@@ -34,10 +28,10 @@ class RankCard extends Component {
           </div>
           <br></br>
           <p style={{ textAlign: "center" }}>
-            Score:{" "}
-            {this.props.team.score === -1000000
+            Rating:{" "}
+            {this.state.rating === -1000000
               ? "N/A"
-              : Math.round(this.props.team.score)}
+              : Math.round(this.state.rating)}
```

Comment:
Let's not "show" ranking, and only show rating? Eg:
<img width="157" alt="image" src="https://user-images.githubusercontent.com/14008996/214677159-0cc3b6ab-3b89-4601-897f-24bb8c896346.png">


(No i don't know the code for this, I faked it with inspect element)